smart-contract/src/contract.rs

```diff
@@ -1739,9 +2010,30 @@ mod tests {
         )
         .unwrap();
 
+        // TODO - fix mock response so it is only returnable for checking bank2 and not bank1
```

Comment:
waiting on resolution from provwasm team about mocking not considering the inputs of the query